platform/platform-app/src/main/resources/features.xml

```diff
@@ -53,128 +53,135 @@
             <!--mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.jaxb-impl/2.2.6_1-->
         <!--</bundle>-->
     <!--</feature>-->
-    <feature name='camel' install="manual" version='2.16.1'  start-level='50'>
+    <feature name='camel' install="manual" version='${camel.version}'  start-level='50'>
         <!-- Added cxf as prereq to keep things ordered -->
         <feature prerequisite="true">cxf</feature>
-        <feature version='2.16.1'>camel-core</feature>
-        <feature version='2.16.1'>camel-spring</feature>
-        <feature version='2.16.1'>camel-blueprint</feature>
-    </feature>
-    <feature name='camel-core' install="manual" version='2.16.1'  start-level='50'>
+        <feature version='${camel.version}'>camel-core</feature>
+        <feature version='${camel.version}'>camel-spring</feature>
+        <feature version='${camel.version}'>camel-blueprint</feature>
+    </feature>
+    <feature name='camel-amqp' version='${project.version}' start-level='50'>
+    <bundle dependency='true'>mvn:commons-lang/commons-lang/${commons-lang.version}</bundle>
+    <bundle dependency='true'>mvn:commons-collections/commons-collections/${commons-collections.version}</bundle>
+    <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.mina/${servicemix.bundles.mina.version}</bundle>
+    <!-- TODO: Specify version range for qpid-jms-client (0.6.0, 0.10.0] once Karaf 4.0.6 is available -->
+    <bundle dependency='true'>wrap:mvn:org.apache.qpid/qpid-jms-client/${qpid-jms-client.version}</bundle>
+    <bundle dependency='true'>wrap:mvn:org.apache.qpid/proton-j/${proton-j.version}</bundle>
+    <bundle dependency='true'>wrap:mvn:io.netty/netty-all/${netty-all.version}</bundle>
+    <feature prerequisite="true" version='${camel.version}'>camel-jms</feature>
```

Comment:
That's correct, these features are copied straight from the camel features file.